WalletWasabi.Fluent/Views/Settings/GeneralSettingsTabView.axaml

```diff
@@ -11,37 +11,37 @@
   <StackPanel Classes="settingsLayout">
 
     <DockPanel>
-      <TextBlock Text="Dark Mode" />
+      <TextBlock Text="Dark mode" />
       <ToggleSwitch IsChecked="{Binding DarkModeEnabled}" />
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Start Wasabi with the system" />
+      <TextBlock Text="Run Wasabi when computer starts" />
       <ToggleSwitch IsChecked="{Binding RunOnSystemStartup}" Command="{Binding StartupCommand}" />
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Hide Window on close" />
+      <TextBlock Text="Allow Wasabi to run in the background" />
```

Comment:
How about `Keep Wasabi running in the background on close`.

Reply:
IMHO, that feels a bit too long.

Reply:
> IMHO, that feels a bit too long.

I agree with that, but the problem is the current text doesn't really tell that this feature will keep Wasabi running even after you close it.

Maybe just `Keep Wasabi running in the background`. I think it is more meaningful than the current one.

Reply:
- `Keep running in system tray`
- `Close Wasabi to system tray`
- `Keep Wasabi in system tray`

As far as I see other applications call this feature somehow like these above.